dp/cloud/go/services/dp/obsidian/models/swagger.v1.yml

```diff
@@ -21,6 +21,9 @@ definitions:
         description: this is the maximum allowed difference in MHz between leftmost end of leftmost channel and rightmost end of rightmost channel used by a Base Station (eNB)
         type: integer
         x-nullable: false
+        maximum: 150
+        minimum: 5
```

Comment:
Can we change it to 20, so that is it always greater than bandwidth of a single channel?

Reply:
Removed `minimum: 5` from swagger validation, but modified `MutableCbsd`'s `ValidateModel` func to ensure that `max_ibw_mhz` is not less than `bandwidth_mhz`.